File: ProgramingBasics/SimpleConditionalStatements/11.EqualWords/src/EqualWords.java
import java.util.Scanner;

public class EqualWords {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input1 = scanner.nextLine().toLowerCase();
        String input2 = scanner.nextLine().toLowerCase();

        if (input1.equals(input2)) {
            System.out.println("yes");
        }
        else {
            System.out.println("no");
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/11.EqualWords/11.EqualWords/EqualWords.cs
using System;

namespace _11.EqualWords
{
    public class EqualWords
    {
        public static void Main(string[] args)
        {
            var input1 = Console.ReadLine().ToLower();
            var input2 = Console.ReadLine().ToLower();

            if (input1 == input2)
            {
                Console.WriteLine("yes");
            }
            else
            {
                Console.WriteLine("no");
            }
        }
    }
}
